Белёв Александр

sanyabas opened the pull request:
[Посмотреть решение](https://rawgit.com/sanyabas/markup-task-5/master/index.html)

sanyabas committed:
task completed

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

mokhov reviewed (changes requested):
1. Все элементы по которым можно кликнуть нужно делать с `cursor: pointer`
2. Когда открыто изображение в фулскрине хорошо бы сделать чтобы клик в любом месте вне изображения закрывал его
3. Тут наехали элементы друг на друга
<img width="377" alt="screen shot 2017-11-08 at 10 07 24" src="https://user-images.githubusercontent.com/357689/32532509-ac20b4a0-c46c-11e7-8f36-87c45ce19f90.png">
🍅

sanyabas committed:
fix after review
pointer cursor on clickable elements
click outside the picture closes it
arrows moved further
fixed page width

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

mokhov commented:
Отлично!
